test/parallel/test-vm-create-and-run-in-context.js

```diff
@@ -1,3 +1,4 @@
+// Flags: --expose-gc
 'use strict';
```

Comment:
nit: other two locations where `--expose-gc` is used the comment is placed directly after the `'use strict';`.